### ISSUE TITLE
Provide ESLint Rules for preferred rxIf/rxFor/push usage over ngIf/ngFor/async

### ISSUE THREAD

**Issue by marvinscharle** [rx-angular/rx-angular#1614]
# Problem Solved By The Feature

When migrating a large codebase over to be zone-less, it would greatly help if not only the ts files can be linted for potential problematic change detection usages, but also the templates as well.

# Solution

Provide rules to prefer:

- `rxIf` over `ngIf`
- `rxFor` over `ngFor`
- `push` over `async`

# alternatives Considered

-

# Additional Context

- 

---

Thank you in advance for considering this feature request and thank you for the great work!

**Comment by BioPhoton** [comment]
I guess we now have to consider the new control flow too :D

**Comment by marvinscharle** [comment]
Yes, I thought about the same. I'd suggest an ESLint rule without auto-fix. Just the hint, that (e.g.) `async` triggers a change detection for all parent view containers as well. 

I'd love to use a rule like this with the warn level.
